.github/workflows/ci.yml

```diff
@@ -190,6 +190,64 @@ jobs:
           name: fasttrackml-archives
           path: dist/*
 
+  build-wheels:
+    needs: golang-build
+    runs-on: ${{ matrix.runner }}
+    timeout-minutes: 20
+    strategy:
+       matrix:
+        os: [darwin, linux, windows]
+        arch: [x86_64, arm64]
+        exclude:
+          - os: windows
+            arch: arm64
+        include:
+          - os: darwin
+            runner: macos-latest
+          - os: linux
+            runner: ubuntu-latest
+          - os: windows
+            runner: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Get binaries
+        uses: actions/download-artifact@v3
+        with:
+          name: fasttrackml-archives
+          path: dist/
+      
+      - name: Configure linux build environment
+        if: matrix.os == 'linux'
+        run: |
+          mkdir -p bin
+          tar -xf dist/fasttrackml_linux_${{ matrix.arch }}.tar.gz
+          mv fml bin/fml
+          chmod +x bin/fml
+      - name: Configure macOS build environment
+        if: matrix.os == 'darwin'
+        run: |
+          mkdir -p bin
+          tar -xf dist/fasttrackml_macos_${{ matrix.arch }}.tar.gz
+          mv fml bin/fml
+          chmod +x bin/fml
+      - name: Configure windows build environment
+        if: matrix.os == 'windows'
+        run: |
+         sudo apt-get install unzip
+         mkdir -p bin
+         unzip dist/fasttrackml_windows_${{ matrix.arch }}.zip -d bin
+         
+      - name: Build wheels
+        run: make python-dist
+        env:
+          GOOS: ${{ matrix.os }}
+          GOARCH: ${{ matrix.arch }}
+
+      - uses: actions/upload-artifact@v3
+        with:
+          name: fasttrackml-wheels
+          path: ./wheelhouse/*.whl
+
```

Comment:
Change `golang-build` to become `build` and run `make dist` that will depend on `python-dist` as well as `go-dist`. This way each os/arch combination will build its own binary and wheel in one go.